akka-remote/src/main/scala/akka/remote/RemotingLifecycleEvent.scala

```diff
@@ -87,6 +87,9 @@ final case class QuarantinedEvent(address: Address, uid: Int) extends RemotingLi
       "from this situation."
 }
 
+@SerialVersionUID(1L)
+final case class ActorSystemQuarantinedEvent(localAddress: Address, remoteAddress: Address)
```

Comment:
Please also extend RemotingLifecycleEvent and add a descriptive `toString` message. In order to distinguish the meaning from QuarantinedEvent this one should be called something like WeHaveBeenQuarantinedEvent (my naming skills are not perfect today, so if you have a better idea … ). I think it should be logged at warning level and replace the old unstructured log message.